.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "parser": "babel-eslint",
```

Comment:
I was kinda using it in the `ava` tests, but not really taking advantage of it, so dropping it.